File: eco-core/core-plugin/src/main/java/com/willfp/talismans/command/CommandReload.java
package com.willfp.talismans.command;

import com.willfp.eco.core.EcoPlugin;
import com.willfp.eco.core.command.CommandHandler;
import com.willfp.eco.core.command.impl.Subcommand;
import org.jetbrains.annotations.NotNull;

public class CommandReload extends Subcommand {
    /**
     * Instantiate a new command handler.
     *
     * @param plugin The plugin for the commands to listen for.
     */
    public CommandReload(@NotNull final EcoPlugin plugin) {
        super(plugin, "reload", "talismans.commands.reload", false);
    }

    @Override
    public CommandHandler getHandler() {
        return (sender, args) -> {
            sender.sendMessage(
                    this.getPlugin().getLangYml().getMessage("reloaded")
                            .replace("%time%", this.getPlugin().reloadWithTime() + "")
            );
        };
    }
}

File: eco-core/core-plugin/src/main/kotlin/com/willfp/talismans/command/CommandReload.kt
package com.willfp.talismans.command

import com.willfp.eco.core.command.impl.Subcommand
import com.willfp.eco.util.StringUtils
import com.willfp.eco.util.toNiceString
import com.willfp.talismans.plugin
import org.bukkit.command.CommandSender

object CommandReload : Subcommand(
    plugin,
    "reload",
    "talismans.command.reload",
    false
) {
    override fun onExecute(sender: CommandSender, args: List<String>) {
        sender.sendMessage(
            plugin.langYml.getMessage("reloaded", StringUtils.FormatOption.WITHOUT_PLACEHOLDERS)
                .replace("%time%", plugin.reloadWithTime().toNiceString())
        )
    }
}